### PR TITLE
Features/activity events

### PR DESCRIPTION
**Pull request by josedlpozo**

### CODE REVIEW

File: galileo/build.gradle [hunk 1]
```diff
@@ -87,6 +87,12 @@ dependencies {
     implementation 'android.arch.lifecycle:extensions:1.1.1'
 
     implementation 'com.squareup:seismic:1.0.2'
+
+    implementation "io.arrow-kt:arrow-core:0.8.0"
+    implementation "io.arrow-kt:arrow-typeclasses:0.8.0"
+    implementation "io.arrow-kt:arrow-instances-core:0.8.0"
+
+    implementation 'com.jakewharton.timber:timber:4.5.1'
```
Comment (vicfran) [inline galileo/build.gradle]: Added again? :D

File: galileo/build.gradle [hunk 2]
```diff
@@ -87,6 +87,12 @@ dependencies {
     implementation 'android.arch.lifecycle:extensions:1.1.1'
 
     implementation 'com.squareup:seismic:1.0.2'
+
+    implementation "io.arrow-kt:arrow-core:0.8.0"
```
Comment (vicfran) [inline galileo/build.gradle]: with arrow hemos topado, what we use it for?

File: galileo/build.gradle (hunk 1 above)
Comment (josedlpozo) [reply galileo/build.gradle]: Merge conflicts problems :dancer:

File: galileo/build.gradle (hunk 2 above)
Comment (josedlpozo) [reply galileo/build.gradle]: I have added it just for testing Arrow! We could use it to improve our functional programming learnings :smile:

**Pull request merged**